### DISCUSSION TITLE
weekly title alternative `2023/10/W41`

### DISCUSSION THREAD

**Discussion by hodanli** [YU000jp/logseq-plugin-show-weekday-and-week-number#117] [Q&A]
hi,

is there anyway that weekly page titles are constructed as 2023/10/W41. this way i can see al weeks in hierarchy. if this is possible within the settings (i could not find) excuse me. if not this would be a nice addition. thanks.

**Comment by YU000jp** [comment]
Its notation is not provided.(https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/discussions/64)  Because there is no common understanding for crossing months, and no way to calculate it. The last week of the month be included either before or after. However, there are no rules for it. For example, only one day is included in the previous month.
